.github/workflows/build-container.yml

```diff
@@ -0,0 +1,53 @@
+name: Release ingress-netperf container image
+
+on:
+  pull_request:
```

Comment:
We must trigger this workflow on a push event. Doing it on a `pull_request` event is dangerous, as any PR would trigger a container build and can introduce incorrect or malicious code.

Reply:
Ack, will change to `pull_request`